src/modules/enrollment/components/dashboardPages/HouseholdPage.tsx

```diff
@@ -13,6 +14,10 @@ const HouseholdPage = () => {
     clientId: string;
   };
 
+  const [householdMembers, { loading: householdMembersLoading, error }] =
```

Comment:
Do you think this is a good approach -- moving the householdMembers query one level up so that HouseholdMemberTable doesn't perform it anymore (and the results can be shared between HouseholdMemberTable and the alerts on the Enrollments page)? Or should HouseholdMemberTable be more ambidextrous and capable of either making the request itself or receiving the household members as props?

Reply:
Let's check in about it next week. From a quick glance, I think it relates to the spaghetti-ness of the new fragment name.

If you load up the Enrollment dashboard from scratch, you should see that all the graphql queries get batched in one request:
<img width="641" alt="Screenshot 2024-01-26 at 4 44 02 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/4f0ae3a1-f8bc-4a5f-b0b3-63dbd771f228">


`GetEnrollmentWithHousehold` is this one thats loading all the members and all the Enrollment details. `GetEnrollmentReminders` loads the "household tasks" reminder card that shows up on the right.

My inclination would be to have the alert card function similarly to the household task card in this case. Meaning, the "Household Alert" card would be responsible for fetching what it needs. It could use a query that _just_ loads the household members' alerts. (A "new" query from the frontend perspective, but using the existing backend household query).  Imo that makes the code a lot more readable -- we can take advantage of GraphQL batching to help us separate concerns.

I know that is sort of contradictory to what I said about the Client Dashboard... on the Client dash, it makes more sense to me to load it on the Client because it's just the one record.

Those are just some initial thoughts! I really don't have a clear-cut answer on this type of question,. and it comes up a lot

Reply:
Yeah I agree with everything you said, thanks for steering me in the right direction!

Reply:
(I'm converting this PR back to draft mode for now haha)